src/python/WMCore/ReqMgr/DataStructs/Request.py

```diff
@@ -70,8 +70,12 @@ def generateRequestName(request):
     current_time = time.strftime('%y%m%d_%H%M%S', time.localtime(time.time()))
     seconds = int(10000 * (time.time() % 1.0))
     request_string = request.get("RequestString", "")
+
     if request_string != "":
         request["RequestName"] = "%s_%s" % (request["Requestor"], request_string)
```

Comment:
Instead changing the fina request name, how about we change this line to:
`request["RequestName"] = "%s_%s" % (request["Requestor"].split('@')[0], request_string)`
?

One more unrelated comment, I'd enforce a RequestString to be provided. Don't ask me for a reason, but it sounds more clear and... organized than having a "amaltaro_160302_1233_43" :-)